src/scripts/main.js

```diff
@@ -1,3 +1,309 @@
+const thead = document.querySelector('thead');
+const headTableRow = thead.querySelector('tr');
+const tbody = document.querySelector('tbody');
+const body = document.querySelector('body');
+
+let countClick = 0;
+const enterKeyCode = 13;
+let initialText = null;
+let selectedCell = null;
+
+const form = document.createElement('form');
+const inputForm = document.createElement('input');
+const inputName = ['name', 'position', 'office', 'age', 'salary'];
+const optionName = [`Tokyo`, `Singapore`, `London`, `New
+York`, `Edinburgh`, `San Francisco`];
+const labelText = ['Name:', 'Position:', 'Office:', 'Age:', 'Salary:'];
+const label = document.createElement('label');
+const select = document.createElement('select');
+const option = document.createElement('option');
+const button = document.createElement('button');
+
+for (let i = 0; i < labelText.length; i++) {
+  label.textContent = labelText[i];
+  form.append(label.cloneNode(true));
+}
+
+for (let i = 0; i < optionName.length; i++) {
+  option.value = optionName[i];
+  option.textContent = optionName[i];
+  select.append(option.cloneNode(true));
+}
+
+for (let i = 0; i < inputName.length; i++) {
+  inputForm.name = inputName[i];
+  inputForm.dataset.qa = inputName[i];
+
+  if (i > 2) {
+    inputForm.type = 'number';
+  }
+
+  if (i === 2) {
+    select.name = inputName[i];
+    select.dataset.qa = inputName[i];
+    form.children[i].append(select.cloneNode(true));
+  } else {
+    form.children[i].append(inputForm.cloneNode());
+  }
+}
+
+button.textContent = 'Save to table';
+form.append(button);
+
+form.classList.add('new-employee-form');
+body.append(form);
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const data = new FormData(form);
+  const inputValues = Object.fromEntries(data.entries());
+  const newTr = document.createElement('tr');
+  const td = document.createElement('td');
+
+  if (!inputValues.position || !inputValues.salary) {
+    notification(
+      'Error',
+      'All fields must be filled',
+      'error'
+    );
+  } else if (inputValues.age > 90 || inputValues.age < 18) {
+    notification(
+      'Error',
+      'The age value should be between 18 and 90 years',
+      'error'
+    );
+  } else if (inputValues.name.length < 4) {
+    notification(
+      'Error',
+      'The name must contain more than 4 letters',
+      'error'
+    );
+  } else {
+    for (const inputValue in inputValues) {
+      if (inputValue === 'name' || inputValue === 'position') {
+        td.textContent = inputValues[inputValue].charAt(0).toUpperCase()
+        + inputValues[inputValue].slice(1, inputValues[inputValue].length);
+        newTr.append(td.cloneNode(true));
+      } else if (inputValue === 'salary') {
+        td.textContent = toCashString(inputValues[inputValue].toString());
+        newTr.append(td.cloneNode(true));
+      } else {
+        td.textContent = inputValues[inputValue];
+        newTr.append(td.cloneNode(true));
+      }
+    }
```

Comment:
try to avoid else if.
if this case suggesting to replace `if ... else if ... else` on `switch...case`.